packages/executor/src/executor.ts

```diff
@@ -106,5 +106,15 @@ export class Executor {
       this.networkConfig,
       this.logger
     );
+
+    if (this.networkConfig.conditionalTransactions) {
+      this.logger.warn(`${this.network}: [x] CONDITIONAL TRANSACTIONS`);
+    }
+
+    if (this.networkConfig.rpcEndpointSubmit) {
```

Comment:
Is this logged as information? If so why only as warn?

Reply:
This is just to make this messages have a different color than the other messages and highlight them.